### PR TITLE
Feat/748

### PR DESCRIPTION
**Pull request by valerydluski**

### CODE REVIEW

File: client/src/pages/course/score.tsx [hunk 1]
```diff
@@ -210,8 +210,49 @@ function renderSettingsModal(
...
+      title="Columns visibility"
+      visible={isVisibleSetting}
+      onOk={onOkHandle}
+      onCancel={handleModalCancel}
+      footer={[
+        <Button
+          key="Select all"
+          onClick={() => {
+            fillAllFields(true);
+          }}
```
Comment (apalchys) [inline client/src/pages/course/score.tsx:234]:
```suggestion
          onClick={() =>  fillAllFields(true)}
```

File: client/src/pages/course/score.tsx [hunk 2]
```diff
@@ -210,8 +210,49 @@ function renderSettingsModal(
...
+          type="text"
+        >
+          Select all
+        </Button>,
+        <Button
+          key="Deselect all"
+          onClick={() => {
+            fillAllFields(false);
+          }}
```
Comment (apalchys) [inline client/src/pages/course/score.tsx:243]:
```suggestion
          onClick={() =>  fillAllFields(false)}
```

**Review by apalchys — APPROVED** [review]

File: client/src/pages/course/score.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/pages/course/score.tsx]: Thanks for the review. Fixed

File: client/src/pages/course/score.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/pages/course/score.tsx]: done

**Pull request merged**